bakery/env/prod.json

```diff
@@ -5,14 +5,14 @@
   "COPS_BUCKET_SAK_SECRET_NAME": "((prod-cops-artifacts-secret-access-key))",
   "COPS_ARTIFACTS_S3_BUCKET": "openstax-cops-artifacts",
   "COPS_CLOUDFRONT_URL": "https://openstax.org",
-  "DIST_ARCHIVE_FEED_FILE_URL": "https://raw.githubusercontent.com/openstax/content-manager-approved-books/master/approved-books.json",
-  "DIST_GIT_FEED_FILE_URL": "https://raw.githubusercontent.com/openstax/content-manager-approved-books/master/approved-git-books.json",
-  "DIST_BUCKET_AKI_SECRET_NAME": "((prod-web-hosting-content-gatekeeper-access-key-id))",
-  "DIST_BUCKET_SAK_SECRET_NAME": "((prod-web-hosting-content-gatekeeper-secret-access-key))",
-  "DIST_S3_BUCKET": "openstax-web-hosting-content-gatekeeper",
-  "DIST_QUEUE_STATE_S3_BUCKET": "openstax-web-hosting-content-queue-state",
-  "DIST_ARCHIVE_QUEUE_FILENAME": "distribution-queue.json",
-  "DIST_GIT_QUEUE_FILENAME": "git-distribution-queue.json",
+  "WEB_ARCHIVE_FEED_FILE_URL": "https://raw.githubusercontent.com/openstax/content-manager-approved-books/master/approved-books.json",
+  "WEB_GIT_FEED_FILE_URL": "https://raw.githubusercontent.com/openstax/content-manager-approved-books/master/approved-git-books.json",
+  "WEB_BUCKET_AKI_SECRET_NAME": "((prod-web-hosting-content-gatekeeper-access-key-id))",
+  "WEB_BUCKET_SAK_SECRET_NAME": "((prod-web-hosting-content-gatekeeper-secret-access-key))",
+  "WEB_S3_BUCKET": "openstax-web-hosting-content-gatekeeper",
+  "WEB_QUEUE_STATE_S3_BUCKET": "openstax-web-hosting-content-queue-state",
+  "WEB_ARCHIVE_QUEUE_FILENAME": "distribution-queue.json",
+  "WEB_GIT_QUEUE_FILENAME": "git-distribution-queue.json",
```

Comment:
Should the queue filenames be updated similar to staging here?

Reply:
Ah yea, they should be. i'll push an update.

Reply:
@rnathuji I've pushed an update. This should be resolved.